org.lflang/META-INF/MANIFEST.MF

```diff
@@ -22,7 +22,8 @@ Require-Bundle: org.eclipse.xtext,
  org.eclipse.lsp4j;bundle-version="0.12.0",
  com.fasterxml.jackson.core.jackson-core,
  com.fasterxml.jackson.core.jackson-annotations,
- com.fasterxml.jackson.core.jackson-databind
+ com.fasterxml.jackson.core.jackson-databind,
+ org.eclipse.core.runtime;bundle-version="3.23.0"
```

Comment:
```suggestion
 org.eclipse.core.runtime;bundle-version="3.24.0"
```
Adding this dependency should be fine. Our Gradle/Maven configurations, however, use `3.24.0` already (see https://github.com/lf-lang/lingua-franca/blob/master/gradle.properties), so we should match it.